### PR TITLE
Initial IST release

### PR DESCRIPTION
**Pull request by rockymontana**

### CODE REVIEW

File: chapters/http-headers.adoc [hunk 1]
```diff
@@ -300,187 +300,4 @@ https://stripe.com/docs/api/idempotent_requests[Stripe API].
...
-
-* Services *must* propagate {Flow-ID}, i.e. use {Flow-ID} received
-with API calls or consumed events as...
-** input for all API called and events published during processing
-** data field written for logging and tracing
-
-*Hint:* This rule also applies to application internal interfaces and events
-not published via Nakadi (but e.g. via AWS SQS, Kinesis or service specific
-DB solutions).
+Zalando landscape specific proprietary headers.
```
Comment (mattiasmelin) [inline chapters/http-headers.adoc]: Should this still say Zalando?

**Review by mattiasmelin — APPROVED** [review]
A quick review and only one comment.

File: chapters/http-headers.adoc (hunk 1 above)
Comment (KhrystynaMakarIST) [inline chapters/http-headers.adoc]: Not sure we should mention it in IST guidelines

File: chapters/http-requests.adoc [hunk 2]
```diff
@@ -493,9 +476,6 @@ provides the following benefits:
 ** No external documents or grammars needed
```
Comment (KhrystynaMakarIST) [inline chapters/http-requests.adoc:481]: Are we going to still use 'Zalando' word in examples? If no, then we need to replace 'Zalando' with something in line 435: * `name=Zalando`, to query for elements based on property equality
_(and maybe in line 52)_

File: chapters/json-guidelines.adoc [hunk 3]
```diff
@@ -53,7 +53,7 @@ of Internet Assigned Numbers Authority (IANA)) as `content-type` (or `accept`) h
 information. More specifically, for JSON payload you should use the standard media type
```
Comment (KhrystynaMakarIST) [inline chapters/json-guidelines.adoc:53]: At the beginning of this document it says:

> "These guidelines provides recommendations for defining JSON data at Zalando.
> JSON here refers to {RFC-7159}[RFC 7159] (which updates {RFC-4627}[RFC 4627]),
> the "application/json" media type and custom JSON media types defined for APIs.
> The guidelines clarifies some specific cases to allow Zalando JSON data to have
> an idiomatic form across teams and services."

Probably 'Zalando' should be replaced with 'IST'

File: chapters/meta-information.adoc [hunk 4]
```diff
@@ -192,7 +191,7 @@ to and a unique a short identifier for the functional component itself:
```
Comment (KhrystynaMakarIST) [inline chapters/meta-information.adoc:191]: Line 186: 

> organizational changes (Zalando for example maintains an internal naming convention).

Just remove everything in () (and parenthesis itself)

**Review by KhrystynaMakarIST — COMMENTED** [review]
I also see here https://ist-group.github.io/rest-guidelines/#101 that some word is missed in line:

> You must the component external API specification with the deployment of the implementing service,...

Probably 'you must publish...' But I can't find this place in a PR :-(

File: chapters/http-requests.adoc (hunk 2 above)
Comment (rockymontana) [reply chapters/http-requests.adoc:481]: Thanks Khrystyna. I couldn't find these places in the code, but I remember specifically changing that query param example to name=IST on Friday, so should be fixed now

File: chapters/meta-information.adoc (hunk 4 above)
Comment (rockymontana) [reply chapters/meta-information.adoc:191]: Should be fixed now

File: chapters/http-headers.adoc (hunk 1 above)
Comment (rockymontana) [reply chapters/http-headers.adoc]: It's fixed

File: chapters/json-guidelines.adoc (hunk 3 above)
Comment (rockymontana) [reply chapters/json-guidelines.adoc:53]: Thanks, I've updated it

**Pull request merged**